apps/frontend/src/app/shared/types/data-store.ts

```diff
@@ -4,4 +4,5 @@ export interface DataStore {
   capacity: number;
   highWaterMark: number;
   filled: number;
+  overflowTime?: number | null;
```

Comment:
```suggestion
  overflowTime?: number;
```
The ? already defines, that it can be undefined